tools/data_loader.py

```diff
@@ -94,21 +94,21 @@ def get_token():
 
         try:
             token = yaml.load(open('.tokens.yaml'), Loader=yaml.Loader)['INITIAL_ADMIN']
-            print('Token loaded from `.tokens.yaml`')
             return token
         except (FileNotFoundError, TypeError, KeyError):
-            print(
-                'Error: no token specified, and no suitable token found in .tokens.yaml'
-            )
             return None
 
     print('Testing connection...', end='')
 
-    RETRIES = 10
+    RETRIES = 15
     timeout = 3
+    admin_token = None
     for i in range(RETRIES):
         try:
+            previous_admin_token = admin_token
             admin_token = get_token()
```

Comment:
```suggestion
            if previous_admin_token = None:
                admin_token = get_token()
```
Otherwise we're still fetching the token for each retry, right?

Reply:
We need to, because the token file changes on disk sometimes.  (I had it the way you suggested first, and that's how I found the 400 authorization errors!)

Reply:
Ah, okay, I was confused because it says this PR makes it so that the token is loaded only once. Perhaps I'm just misunderstanding that part, apologies!

Reply:
Durnit; Github grabbed that from my commit message, but of course it doesn't reflect that I updated the commit message!  Sorry, I will edit it.